README.md

```diff
@@ -45,6 +45,14 @@ The software is useful for at least these scenarios:
 Version 0.2.0 will have a Web UI, which will probably make the whole
 thing much more enjoyable.
 
+## Wiki
+
+For more information please check out the git hub wiki : https://github.com/e14n/pump.io/wiki
```

Comment:
It'd be great if a) this said "GitHub" instead of "git hub" and b) this was a Markdown link, instead of just a URL.